pkg/providers/aws/cluster_vpc.go

```diff
@@ -511,3 +526,7 @@ func getSecurityGroup(ec2Svc ec2iface.EC2API, secName string) (*ec2.SecurityGrou
 
 	return foundSecGroup, nil
 }
+
+func getPrefixedClusterID(clusterID string) string{
```

Comment:
`getOSDClusterTagKey` might be a more descriptive name for this function. Its confusing to read in places that its called in conditionals.

Reply:
renamed (never were able to give a good name to anything in my code :disappointed: )